org.eclipse.winery.repository/src/main/webapp/jsp/servicetemplates/boundarydefinitions/boundarydefinitions.jsp

```diff
@@ -499,7 +491,6 @@
 						contentType: "application/json"
 					}).done(function (result) {
 						vShowSuccess("Sucessfully created interface");
-						updateInterfaces();
```

Comment:
Why is this removed? If I change the interface from `x` to `y`, the operations of `y` have to be shown, not the one of `x` kept.